### PR TITLE
Update README.md and docs

### PR DESCRIPTION
**Pull request by al-fisher**
### Issues addressed by this PR

Closes partially https://github.com/BHoM/admin/issues/4

### Changelog
- Update of README.md
- Update of CONTRIBUTING.md
- Update of GOVERNANCE.md

### CODE REVIEW

**Comment by al-fisher** [comment]
Included small tweak in last commit to governance wording based on @adecler comments

**Comment by rwemay** [comment]
LGTM, I think worth getting some comments from some of the people you've spoken with recently @al-fisher , if they're in the org/keen to have a look.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,68 +1,50 @@
 # Contributing to the BHoM
 
-Geat to see you here! :smile: There are a number of ways to contribute and get involved. Please take a read and do reach out if curious to join in with our experiment. [hello@bhom.xyz](mailto:hello@bhom.xyz)  
+Great to see you here! :smile: There are a number of ways to contribute and get involved in the Buildings and Habitats object Model. Please do take a read and reach out if curious to join in with our collective experiment. [hello@bhom.xyz](mailto:hello@bhom.xyz)
```
Comment (FraserGreenroyd) [inline CONTRIBUTING.md:3]: For people coming to this document for the first time, the header is `Contributing to the BHoM` while the intro is `get involved in the Building and Habitats object Model`. I would advise we change the intro to read:
`get involved in the Building and Habitats object Model (the BHoM)` to provide the first introduction to the acronym of BHoM.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,68 +1,50 @@
...
 
+Do also take a look at our [code of conduct](https://github.com/BHoM/BHoM/blob/master/docs/CODE_OF_CONDUCT.md) (_we also have one for [bots too](https://github.com/BHoM/BHoM/blob/master/docs/CODE_OF_CONDUCT_FOR_BOTS.md) if that is you?_ :+1: :robot:)
+
 
 ## Get stuck in
 
 - Submit an issue
 
-  Found a bug? Have an idea for a feature? Why not [submit an issue](https://github.com/BHoM/documentation/wiki/Submitting-an-Issue)?
+  Found a bug? :bug: Have an idea for a feature? :brain: Why not [submit an issue](https://github.com/BHoM/documentation/wiki/Submitting-an-Issue)?
```
Comment (FraserGreenroyd) [inline CONTRIBUTING.md:13]: Is it worth including the `have a question` option of issue raising here?

File: README.md [hunk 3]
```diff
@@ -1,55 +1,32 @@
...
+If you are new (_welcome!_  👋 😄)  a great place to start is on [bhom.xyz](https://bhom.xyz) or reading our [wiki here](https://github.com/BHoM/documentation/wiki) including pages like the [Structure of the BHoM](https://github.com/BHoM/documentation/wiki/Structure-of-the-BHoM) and [Using the BHoM](https://github.com/BHoM/documentation/wiki/Using-the-BHoM).
 
-Try the [installer](http://bhom.xyz/) and a selection of [sample scripts](https://github.com/BHoM/samples).
+This particular repo contains all of the core _object Model_ (oM) definitions. Our community curated common language.
 
+The objects themselves are designed to be very simple, and therefore flexible and customisable. Comprising of nothing more than a list of defining properties. See [defining objects in the oM](https://github.com/BHoM/documentation/wiki/BH.oM-%E2%80%90-Define-New-Objects) for more details.
 
+The core BHoM contains objects from wide ranging domains across the desciplines of engineering, architecture, our environment and the physical world, as well as defining concepts within programming, machine learning, mathematics, geometry and planning to name a few.
```
Comment (FraserGreenroyd) [inline README.md:14]: Sp - `disciplines` rather than `desciplines`?

File: docs/GOVERNANCE.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
 # BHoM Governance 
 
 
-The development of the BHoM is led by a number of core maintainers within the BHoM Organisation. These individuals and their focuses are listed below.
+The development of the Buildings and Habitats object Model is led by a number of core maintainers within the BHoM Organisation. These individuals and their focuses are listed below.
```
Comment (FraserGreenroyd) [inline docs/GOVERNANCE.md:4]: See previous comment about including `(the BHoM)` in the first line for people coming to this document first

File: docs/GOVERNANCE.md [hunk 5]
```diff
@@ -33,38 +33,33 @@ Typical activities of a Repository Maintainer include:
 * Supporting both testing and any releases of code
 
 
-The repositories and/or individual namespaces are summarised below. However, for more detail on individual responsibilities for code within repos please refer to the local CODEOWNERS documents.
+A number of the key repositories and/or individual namespaces are summarised below. However for more detail on individual responsibilities for code within repos please refer to the local CODEOWNERS documents.
```
Comment (FraserGreenroyd) [inline docs/GOVERNANCE.md:36]: Is the reference to `local CODEOWNERS` valid given our decision that only BHoM and BHoM_Engine and select few repos will contain `codeowners` files?

File: docs/GOVERNANCE.md [hunk 6]
```diff
@@ -33,38 +33,33 @@ Typical activities of a Repository Maintainer include:
...
 * Structure oM/Engine  &nbsp;  @IsakNaslundBh  
-* Environment oM/Engine  &nbsp;  @FraserGreenroyd @michaldengusiak
-
+* Environment oM/Engine  &nbsp;  @FraserGreenroyd
+* Geometery & BIM &nbsp;  @pawelbaran 
+* Data oM & Machine Learning &nbsp; @epignatelli  
+* BHoM_Adapter & Interoperability &nbsp; @alelom
+* Structural_Adapters  &nbsp;  @IsakNaslundBh  
+* Environmental_Adapters  &nbsp;  @FraserGreenroyd   
+* Test_Toolkit & Continuous Integration &nbsp;  @FraserGreenroyd
```
Comment (FraserGreenroyd) [inline docs/GOVERNANCE.md:52]: Would it not be better to put this the other way round as `Continuous Integration & Test_Toolkit` given Test_Toolkit is a subset of Continuous Integration and Continuous Integration is the bigger element?

**Comment by adecler** [comment]
LGTM taking into account the corrections from @FraserGreenroyd. It is starting to be a bit too much of emoji overload for my liking but I can live with it :wink:

File: CONTRIBUTING.md (hunk 2 above)
Comment (al-fisher) [reply CONTRIBUTING.md:13]: Will add some more text to the community and questions section to refer to questions issues too.

File: CONTRIBUTING.md (hunk 1 above)
Comment (al-fisher) [reply CONTRIBUTING.md:3]: Yep! Good idea

File: docs/GOVERNANCE.md (hunk 4 above)
Comment (al-fisher) [reply docs/GOVERNANCE.md:4]: yep - added

File: docs/GOVERNANCE.md (hunk 5 above)
Comment (al-fisher) [reply docs/GOVERNANCE.md:36]: Will tweak to refer to only the core oM and Engine

File: docs/GOVERNANCE.md (hunk 6 above)
Comment (al-fisher) [reply docs/GOVERNANCE.md:52]: Nice one. Also agreed 😄

**Comment by al-fisher** [comment]
Thanks @FraserGreenroyd, guys for comments!

If you're happy @FraserGreenroyd, we'll get it merged I think.
@rwemay I will share links to this once on master to a few guys 👍 

p.s. @adecler think I am approaching even my limit of emojis - so you can sleep soundly 😴🛏️

🤣

**Pull request merged**